prime-router/src/main/kotlin/fhirengine/engine/FHIRReceiverFilter.kt

```diff
@@ -357,6 +357,7 @@ class FHIRReceiverFilter(
                         ),
                         metadata = this.metadata,
                         topic = queueMessage.topic,
+                        destination = receiver,
```

Comment:
We are assuming this change will fix https://github.com/CDCgov/prime-reportstream/issues/15645? Can we please confirm that and, if so, update this PR description to link to fixing that ticket?

Reply:
We don't know if that fixes it or not.

Reply:
This did not fix either bug (though it is required to fix them). More details to come but I think we should merge this and continue fixing the bugs in a separate PR.